<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExplainWindow</class>
 <widget class="QMainWindow" name="ExplainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/pages/슬라이드2_2.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="move_previous">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>810</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="move_next">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>810</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
